completions/ssh

```diff
@@ -550,13 +550,13 @@ _comp_xfunc_scp_compgen_local_files()
         _comp_compgen -RU files split -l -- "$(
             command ls -aF1dL "${files[@]}" 2>/dev/null |
                 command sed -e "s/$_comp_cmd_scp__path_esc/\\\\&/g" \
-                    -e '/[^\/]$/d' -e "s/^/${1-}/"
+                    -e '/[^/]$/d' -e "s/^/${1-}/"
         )"
     else
         _comp_compgen -RU files split -l -- "$(
             command ls -aF1dL "${files[@]}" 2>/dev/null |
                 command sed -e "s/$_comp_cmd_scp__path_esc/\\\\&/g" \
-                    -e 's/[*@|=]$//g' -e 's/[^\/]$/& /g' -e "s/^/${1-}/"
+                    -e 's/[*@|=]$//g' -e 's/[^/]$/& /g' -e "s/^/${1-}/"
```

Comment:
A test for the local-path generation without `-d` has the same issue. We cannot check the trailing space within the current test framework.

Reply:
This is revisited in #1408.